code/modules/projectiles/guns/energy/special.dm

```diff
@@ -112,11 +110,10 @@
 	name = "energy crossbow"
 	desc = "A reverse engineered weapon using syndicate technology."
 	icon_state = "crossbowlarge"
-	w_class = WEIGHT_CLASS_NORMAL
+	w_class = WEIGHT_CLASS_BULKY
```

Comment:
does this fit on any slot or do you have to carry it on your belt

Reply:
It should fit into armor vests